src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.ts

```diff
@@ -458,7 +458,7 @@ export class TableComponent implements AfterContentChecked, OnInit, OnChanges, O
     // Ensure we do not process DOM 'select' events.
     // https://github.com/swimlane/ngx-datatable/issues/899
     if (_.has($event, 'selected')) {
-      this.selection.selected = $event['selected'];
+      this.selection.update();
```

Comment:
I think we better should do this because it is not guaranteed that `this.selection.selected` already contains the selected rows.
```suggestion
      this.selection.selected = $event['selected'];
      this.selection.update();
```

Reply:
Done.